app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -51,7 +51,7 @@
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
                 android:layout_marginStart="24dp"
-                android:layout_marginTop="10dp"
+                android:layout_marginTop="7dp"
                 android:layout_marginEnd="24dp"
                 android:ems="10"
                 android:hint="@string/user_last_name"
```

Comment:
fix the 'autofillHints' attribute and try not to ignore it

Reply:
ok, done